internal/services/admin/v1/server.go

```diff
@@ -540,6 +540,21 @@ func getCreateWorkflowOpts(req *contracts.CreateWorkflowVersionRequest) (*v1.Cre
 		cronInput = []byte(*req.CronInput)
 	}
 
+	defaultFilters := make([]v1.DefaultFilter, 0)
+	for _, f := range req.DefaultFilters {
+		var payload []byte
+
+		if f.Payload != nil {
+			payload = []byte(*f.Payload)
```

Comment:
There should be a check that this is valid JSON and a user-friendly error if it's not

Reply:
good call, will fix that